doc/guides/nest2_to_nest3/nest2_to_nest3_overview.rst

```diff
@@ -566,6 +566,53 @@ Iterator of sources and targets
     >>>  print([s*3 for s in synColl.sources()])
          [3, 3, 6, 6]
 
+.. _collocated_synapses
+
+Collocated synapses
+~~~~~~~~~~~~~~~~~~~
+It is now possible to create connections with several synapses simultaneously. The different synapse dictionaries will
+then be applied to each source-target pair. To create these collocated synapses ``CollocatedSynapses()`` must be used
+as the `syn_spec` argument of ``Connect``, instead of the usual syn_spec *dictionary*. ``CollocatedSynapses()`` takes
```

Comment:
The italicization does not seem quite right here. We are trying to emphasize that it is the "argument" now as opposed to the "dictionary", right?

Reply:
@sarakonradi I am trying to emphasize that it is a `CollocatedSynapse()` object that is used as argument instead of the dictionary that is usually used as an argument.
Should I remove the italic, and maybe add an "argument" after dictionary? So that the sentence becomes:

To create these collocated synapses, ``CollocatedSynapses()`` must be used as the `syn_spec` argument of ``Connect``, instead of the usual syn_spec dictionary argument.

Reply:
Yes, this version is much clearer!